instrumentation/opentelemetry-instrumentation-psycopg/tests/test_psycopg_integration.py

```diff
@@ -245,14 +245,18 @@ def test_span_name(self):
         cursor.execute("/* leading comment */ query")
         cursor.execute("/* leading comment */ query /* trailing comment */")
         cursor.execute("query /* trailing comment */")
+        cursor.execute("")
+        cursor.execute("--")
```

Comment:
@xrmx I'm adding this empty comment string in the test case as well. Not sure if the expected value (`--`) is correct though.